src/admin/dashboard/pages/settings/Elements/Fields/Commission/CategoryBasedCommission.tsx

```
@@ -0,0 +1,575 @@
+    };
+
+    const showCatRow = ( item ) => {
+        if ( Number( item?.parent_id ) === 0 ) {
+            return true;
+        }
+
+        return openRows.includes( Number( item?.parent_id ) );
+    };
+
+    const getCommissionValue = ( commissionType, termId ) => {
+        if ( commission?.items?.hasOwnProperty( termId ) ) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)